### DISCUSSION TITLE
How to use @enable3d/ammo-on-nodejs with typescript?

### DISCUSSION THREAD

**Discussion by FlokiTV** [enable3d/enable3d#330] [Help / Questions]
Hello guys!
I'm trying to setup a simple scene to test `@enable3d/ammo-on-nodejs`.
I copy the example from https://enable3d.io/examples/headless-mode.html
I replace the `require` for `import`

```ts
import * as _ammo from "@enable3d/ammo-on-nodejs/ammo/ammo.js";
import * as pkg from "@enable3d/ammo-on-nodejs";
const { Physics, ServerClock } = pkg;
```

and when I'll create a `new Physics()` I received this error

```
TypeError: Physics is not a constructor
```

I tryed this and works

```ts
import pkg from "@enable3d/ammo-on-nodejs";
const { Physics, ServerClock } = pkg;
```

but on the first line, I received this warning

```
Module '"[...]/node_modules/@enable3d/ammo-on-nodejs/dist/index"' has no default export.
```

and to load Ammo.js I needed this trick

```ts
import * as _ammo from "@enable3d/ammo-on-nodejs/ammo/ammo.js";
// wait for Ammo to be loaded
_ammo.default().then((ammo: any) => {
  global.Ammo = ammo;
  new ServerScene();
});
```

Has a better way to do this with Typescript?

**Comment by yandeu** [comment]
Just made an example repository:
https://github.com/enable3d/ammo-on-nodejs-example
